tests/ui/pattern/usefulness/floats.rs

```diff
@@ -1,4 +1,6 @@
 #![deny(unreachable_patterns)]
```

Comment:
These will get executed as well. You can make the whole test `//@ check-pass`

Reply:
Thanks for the catch, changed